qa-test/src/bin/embassy_i2s_read.rs

```diff
@@ -75,11 +75,17 @@ async fn main(_spawner: Spawner) {
         println!("available {}", avail);
 
         let count = transaction.pop(&mut data).await.unwrap();
+
+        #[cfg(not(feature = "esp32s2"))]
         println!(
             "got {} bytes, {:x?}..{:x?}",
             count,
             &data[..10],
             &data[count - 10..count]
         );
+
+        // esp-println is a bit slow on ESP32-S2 - don't run into DMA too late errors
+        #[cfg(feature = "esp32s2")]
+        println!("got {} bytes", count,);
```

Comment:
Maybe lets just use the simplified print always, unless printing the bytes is required as part of the qa test?

Reply:
I usually look at the received bytes when e.g. connecting GND,3V3 or WS to DIN (just to see we are actually receiving anything)